src/components/breadcrumbs/Breadcrumbs.test.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import "@testing-library/jest-dom";
+import { render } from "@testing-library/react";
+import { Route } from "react-router";
+
+import Breadcrumbs from "./Breadcrumbs";
+import RoutingTestWrapper from "../../tests/RoutingTestWrapper";
+
+describe("Breadcrumb Component", () => {
+  it("breadcrumbs renders correctly", async () => {
+    const route = "/";
+    const { getByTestId } = render(
+      <RoutingTestWrapper path={route}>
+        <Route path={route} element={<Breadcrumbs styling="" />} />
+      </RoutingTestWrapper>
+    );
+
+    expect(getByTestId("breadcrumbs")).toBeInTheDocument();
+  });
+
+  it("should have correct test as determined by the route", () => {
+    const route = "/NutritionalInformation";
+    const { getByText } = render(
```

Comment:
The RoutingTestWrapper is a bit of a naive wrapper around MemoryRouter. It could do with being refactored to pass more/different values. Currently it only takes a single path but typically you would pass a whole array.

I think also worth adding a bit more to the route to check there are multiple bits of breadcrumb rendered

```suggestion
    const route = "/NutritionalInformation/Salmon";
    render(
      <MemoryRouter initialEntries={["/", route]}>
        <Routes>
          <Route path={route} element={<Breadcrumbs styling="" />} />
        </Routes>
      </MemoryRouter>
    );
    expect(screen.getByText("Food & Nutrition")).toBeVisible();
    expect(screen.getByText("Salmon")).toBeVisible();
```

Reply:
I've used screen here rather than doing object restructuring from the render method. It doesn't really matter here but makes things a bit cleaner when you're using multiple different types of get/find/query selectors.

Reply:
Added this test in to cover multiple routes